### DISCUSSION TITLE
Is possible to change hotkey for search bar?

### DISCUSSION THREAD

**Discussion by sheldygg** [fuma-nama/fumadocs#351] [Q&A]
I want change hotkey for search bar? Is possible? or this required changes in your library?

**Comment by fuma-nama** [comment]
Impossible without change to the codebase

**Reply by sheldygg** [reply]
I think it will be comfortable if your library support this feature.

**Reply by fuma-nama** [reply]
The problem is how we design the API. The key press handler could not be overridden because you will need to interact with internal react contexts to open the search dialog, this is not good in my opinion. 
And if we allow passing the hot keys directly, how could we express complex compounds like "Shift + A" in both UI and the key press handler? I will avoid using other packages as it feels redundant to include a package into the client bundle for this.
I will look for some possible solutions, perhaps we can support it next minor release

**Reply by Shubham-EV** [reply]
I think the search bar would look nice if it had a loading indicator while searching for items because, on first input, it takes some time to load. Are you considering adding that?
